sinks/authentication_type/basicauth/authentication.go

```diff
@@ -35,8 +38,8 @@ var (
 const AuthType = "basicauth"
```

Comment:
nit: for consistency with `bearertokenauth`, AuthType constant could be moved into first consts block alongside with username and password